README.md

```diff
@@ -56,8 +56,6 @@ For now, it only runs on Python 2.7 (not 3.4+).
    [Twilio Account Settings](https://www.twilio.com/user/account/settings).
    You will also need a `TWILIO_NUMBER`, which you may find [here](https://www.twilio.com/user/account/phone-numbers/incoming).
 
-   Run `source .env` to export the environment variables.
-
 1. Make sure the tests succeed.
 
     ```bash
```

Comment:
the `coverage` step wont work without instructions on how to install it. alteratively, we could just run the tests using `python3`

Reply:
Coverage works out of the box if the dependencies from requirements.txt are installed. But for the sake of simplifying the instructions, I removed coverage here and use only python.

Reply:
oh weird - it didnt work for me for some reason. but cool, this is simpler.